packages/grid/helm/syft/values.yaml

```
@@ -132,6 +133,7 @@ proxy:
 # =================================================================================
 
 registry:
+  enabled: false
```

```suggestion
  enabled: true
```
We could have this as enabled by default, 
because we the use internal registry as an intermediate cache, if people are building first and then pushing later.
For customized deployments , we could disable it by values.yaml